x-pack/plugins/watcher/server/routes/api/register_list_fields_route.ts

```diff
@@ -15,22 +15,21 @@ const bodySchema = schema.object({
   indexes: schema.arrayOf(schema.string()),
 });
 
-function fetchFields(dataClient: ILegacyScopedClusterClient, indexes: string[]) {
-  const params = {
-    index: indexes,
-    fields: ['*'],
-    ignoreUnavailable: true,
-    allowNoIndices: true,
-    ignore: 404,
-  };
-
-  return dataClient.callAsCurrentUser('fieldCaps', params);
+function fetchFields(dataClient: IScopedClusterClient, indexes: string[]) {
+  return dataClient.asCurrentUser.fieldCaps(
+    {
+      index: indexes,
+      fields: ['*'],
+      allow_no_indices: true,
```

Comment:
Looks like the `ignoreUnavailable` param was omitted. Was that deliberate?

Reply:
Nope this was a mistake!